index.html

```diff
@@ -15,6 +15,11 @@
 <p>
   Fork me? Fork you, @octocat!
 </p>
+<p>
+  Never fear,
```

Comment:
This is a test for me to see how to make a comment on a pull request already submitted